ee/dtr/admin/install/index.md

```diff
@@ -54,99 +58,68 @@ your profile on the left navigation pane, and select **Admin Settings > Docker T
         --ucp-url <ucp-url>
       ```
 
-    If you want to point this parameter to a load balancer that uses HTTP for
-    health probes over port `80` or `443`, temporarily reconfigure the load
-    balancer to use TCP over a known open port. Once DTR is installed, you can configure the load balancer however you need to.
+    To point the parameter to a load balancer that uses HTTP for health probes over port `80` or `443`, temporarily reconfigure the load balancer to use TCP over a known open port. Once DTR is installed, you can configure the load balancer as necessary.
 
-3. Run the DTR install command on any node connected to the UCP cluster, and with the Docker Engine installed. DTR will not be installed on the node you run
-the install command on. DTR will be installed on the UCP worker defined by the
-`--ucp-node` flag.
+4. Run the DTR install command on any node connected to the UCP cluster on which the Docker Engine is installed. Note that DTR will not be installed on the node on which the install command is run, but on the UCP worker defined by the `--ucp-node` flag. For example, it is possible to SSH into a UCP node and run the DTR install command from there. When running the installation command in interactive TTY or `-it` mode, you will be prompted for any required additional information. [Learn more about installing DTR](/reference/dtr/2.7/cli/install/).
 
-    For example, you could SSH into a UCP node and run the DTR install command
-    from there.  Running the installation command in interactive TTY or `-it` mode
-    means you will be prompted for any required additional information.  [Learn more about installing DTR](/reference/dtr/2.7/cli/install/).
+    * To install a specific version of DTR, replace `{{ page.dtr_version }}` with the  desired version in the [installation command](#step-3-install-dtr). (All DTR versions can be found on the [DTR release notes](/ee/dtr/release-notes/) page.)
 
-    Here are some useful options you can set during installation:
-    * To install a specific version of DTR, replace `{{ page.dtr_version }}` with
-    your desired version in the [installation command](#step-3-install-dtr)
-    above. Find all DTR versions in the [DTR release notes](/ee/dtr/release-notes/)
-    page.
+    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
```

Comment:
This sentence is not clear. Should "it" be deleted?

Reply:
No. As seen in the original bullet, what is happening is that due to DTR being deployed with self-signed certificates UCP may not be able to pull images **from** it. Correction made.